src/VsixV3/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.Managed.vsmanproj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\build\Targets\ProducesNoOutput.Settings.targets" />
+  <Import Project="..\..\..\build\Targets\VSL.Versions.targets" />
+
+  <PropertyGroup>
+    <FinalizeManifest>true</FinalizeManifest>
+    <FinalizeSkipLayout>true</FinalizeSkipLayout>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <OutputPath>$(OutDir)\VsixV3</OutputPath>
+    <IsPackage>true</IsPackage>
+  </PropertyGroup>
+
+  <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.props" />
+  <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.targets" />
+
+  <Target Name="BeforeBuild">
+    <MSBuild Projects="Microsoft.VisualStudio.ProjectSystem.Managed.swixproj" Targets="Build" />
+    <MSBuild Projects="Microsoft.VisualStudio.ProjectSystem.CommonFiles.swixproj" Targets="Build" />
+  </Target>
+
+  <ItemGroup>
+    <MergeManifest Include="$(OutputPath)\Microsoft.VisualStudio.ProjectSystem.Managed.json" />
+    <MergeManifest Include="$(OutputPath)\Microsoft.VisualStudio.ProjectSystem.CommonFiles.json" />
```

Comment:
Perhaps this should be *.ProjectSystem.Managed.CommonFiles.*, though that is a long name!